exchange/websocket/connection.go

```diff
@@ -56,15 +56,17 @@ type Connection interface {
 	IncomingWithData(signature any, data []byte) bool
 	// MatchReturnResponses sets up a channel to listen for an expected number of responses.
 	MatchReturnResponses(ctx context.Context, signature any, expected int) (<-chan MatchedResponse, error)
+	// SubStore returns the subscription store for the connection
+	SubStore() *subscription.Store
```

Comment:
🦍 Optional pedantic nitpick:
```suggestion
	// Subscriptions returns the subscription store for the connection
	Subscriptions() *subscription.Store
```